### PR TITLE
[Action Recognition] Enable extra augmentations

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

File: pytorch_toolkit/action_recognition/action_recognition/spatial_transforms.py [hunk 1]
```diff
@@ -559,8 +637,16 @@ def __repr__(self):
     # expects float image
     def __call__(self, image):
         if self.rnd:
-            enhancer = ImageEnhance.Contrast(image)
-            image = enhancer.enhance(self.factor)
+            if isinstance(image, np.ndarray):
```
Comment (snosov1) [inline pytorch_toolkit/action_recognition/action_recognition/spatial_transforms.py:640]: Is there any particular reason to allow for 2 possible types?

Comment (druzhkov-paul) [reply pytorch_toolkit/action_recognition/action_recognition/spatial_transforms.py:640]: It was designed in such a way. Some transforms expect to have `numpy.ndarray`, others operate with `PIL.Image`.

Comment (snosov1) [reply pytorch_toolkit/action_recognition/action_recognition/spatial_transforms.py:640]: Well, it has a certain smell to it. I'd expect the implementation to use a single image processing lib throughout. But, I guess, it doesn't make sense to remove the implemented paths now. Out of curiosity - is OpenCV or PIL preferrable (from perf. or an other perspective)?

Comment (druzhkov-paul) [reply pytorch_toolkit/action_recognition/action_recognition/spatial_transforms.py:640]: Have no single answer here. PIL is the default choice for TorchVision. I believe that OpenCV should be faster on its own, but not sure if this is 100% true given the whole data loading pipeline.

**Review by snosov1 — APPROVED** [review]

**Pull request merged**